### DISCUSSION TITLE
Data Exporting to File

### DISCUSSION THREAD

**Discussion by zwanghi** [OpenDrift/opendrift#1354] [Q&A] [answered]
Hi,

I wanna export the Simulaton data to a csv file or any other maching reading format (netcdf). Is there an easy way to do this?
I would need the data from each particle like latitude, longitude, depth, status, state,....
I tried to export o.elements but somehow it didnt work. Any ideas?

**Comment by knutfrode** [comment, marked as answer]
If you add `outfile=<filename.nc>` to the `run` method, all information will be saved to a CF-compliant netCDF file.

Please see the tutorial: https://opendrift.github.io/tutorial.html#running-the-model

**Reply by zwanghi** [reply]
Thanks a lot, missed it in the docs.
